runtime/doc/change.txt

```diff
@@ -986,10 +986,10 @@ inside of strings can change!  Also see 'softtabstop' option. >
 
 							*Y*
 ["x]Y			yank [count] lines [into register x] (synonym for
-			yy, |linewise|).  If you like "Y" to work from the
-			cursor to the end of line (which is more logical,
-			but not Vi-compatible) use ":map Y y$".
-
+			yy, |linewise|). This is the builtin mapping, see
+			|Y-default| for neovim's default mapping.
+							*Y-default*
+["x]Y                   This command maps to "y$". see |nvim-default-mappings|
```

Comment:
- I do not understand why the `["x]Y` in column 1 is repeated.
- Don't need "see".
- Can be more terse without loss of clarity, and now we have a repeatable pattern.

```suggestion
["x]Y                   Maps to "y$" by default. |nvim-default-mappings|
```